### ISSUE TITLE
I modified class DFL(nn.Module) in the block.py file

### ISSUE THREAD

**Issue by whxuexi** [ultralytics/ultralytics#15644]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I modify the class DFL(nn.Module) in the block.py file to return self.conv(x.view(b, 4, self.c1, a).transpose(2, 1).softmax(1)).view(b, 4, a) "change to" return self.conv(x.view(b, self.c1, 4, a).soft
